bundles/EcommerceFrameworkBundle/Resources/views/admin_order/detail.html.twig

```diff
@@ -213,7 +214,9 @@
                                     <small>
                                         {% set amount = null %}
                                         {% set break = false %}
-                                        {% for providerKey, providerClass in paymentProviders if not break %}
+                                        {% for providerKey, providerClass in paymentProviders %}
+                                            {% if not break %}
```

Comment:
@fashxp I think the if block should be over the whole for block

Reply:
thx, good catch!